Lusas_Adapter/Convert/ToBHoM/Loads/ToBarDistributedLoad.cs

```diff
@@ -87,8 +93,9 @@ public static partial class Convert
                 endPosition,
                 endForceVector,
                 endMomentVector,
-                LoadAxis.Local,
-                false,
+                relativePositions,
+                axis,
+                projected,
                 GetName(lusasBarDistributedLoad));
 
```

Comment:
After creating the object there should be a null check because there are a number of conditions at the start of the `Create` method that can produce a null - this produces an error when trying to add the `AdapterId` as a fragment.